grpc.gemspec

```diff
@@ -1655,6 +1655,99 @@ Gem::Specification.new do |s|
   s.files += %w( third_party/cares/config_freebsd/ares_config.h )
   s.files += %w( third_party/cares/config_linux/ares_config.h )
   s.files += %w( third_party/cares/config_openbsd/ares_config.h )
+  s.files += %w( third_party/libuv/include/uv.h )
```

Comment:
I don't understand how ruby is going to work when other languages (python, C++ etc) have to go out of their way to build the right subset of the sources for given platform (uv_linux, uv_darwin, un_linux), but ruby is just including a list of source files regardless of the platform. So it seems that either a) the platform specific libraries aren't necessary b) ruby won't work.
I think PHP might be a similar story?